src/pheval/utils.py

```diff
@@ -1,7 +1,92 @@
 """
 Contains all pheval utility methods
 """
+import itertools
+from os import path
+from typing import List
 
+import pandas as pd
+from click import Path
 
-def example():
-    return True
+
+def ensure_file_exists(*files: str):
+    """ensure_file_exists"""
```

Comment:
Plese provide better descriptions of what each method does.

Reply:
Also, please use pydocs to document all the parameters in each method you write.